trymath/strings-module.js

```diff
@@ -0,0 +1,28 @@
+const firstChar = (string) => {
+    return string[0];
+}
+
+
+const compare = (stringA, stringB) => {
+    let bool;
+    stringA === stringB ? bool = true : bool = false;
```

Comment:
this can be made even shorter by...


`return stringA === stringB ? true : false;`